### DISCUSSION TITLE
WinEdtPDF does not handle files correctly whose names contain an emoji

### DISCUSSION THREAD

**Discussion by tozGitHub** [WinEdt-Team/WinEdt-Support#26] [2.2 Issues, Bugs and Problems]
Dear Alex, dear community,

I tried using the inverse search in WinEdtPDF with a file called 🎃.tex (the pumpkin emoji), but it failed:
<img width="266" alt="Pasted image 20240218144518" src="https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/3fd5a2a6-0d15-4c62-9ee2-f8f4d910c3a9">
I think the file name shown in the box is strange.
It is not that I ever needed to have emojis in the name of my TeX-files, but I was wondering if they work.

WinEdt versions tested: Build: 20230519  (v. 11.1) - 64-bit and Build: 20240319  (v. 1.2) - 64-bit (the portable one)
Windows version: 10.0.22631 Build 22631.

Best,
Thomas

**Comment by Alex-Simonic** [comment]
I was curious about this one. Both WinEdt and WinEdtPDF handle your 🎃.tex filename fine and TeX compiles it OK from WinEdt. But when I opened 🎃.synctex file it failed to load in UTF8 and the name in synctex was displayed as "í ¼í¾ƒ.tex" instead of "🎃.tex". The same happened when I opened the file in Notepad. The synctex file is NOT in UTF8 encoding (or any other encoding or code page that I tried). These are the first two lines in "🎃.synctex" as pasted from Notepad (or WinEdt):

```
SyncTeX Version:1
Input:1:C:\Alex\Temp\í ¼í¾ƒ.tex
```
I have no idea how to interpret this filename. If this is a bug it is in TeX's synctex generator or else it is some UTF8 sequence that is not recognized or supported by Windows. In either case, it is not a WinEdt problem and there is nothing I can do about it. When I tried Unicode (Chinese) characters in the filename it worked fine. The synctex file was in UTF8 and the inverse search worked fine.
